CoreErrorReportView/src/au/gov/asd/tac/constellation/views/errorreport/ErrorReportTopComponent.java

```diff
@@ -521,39 +527,80 @@ private void updateSettings() {
+        final boolean warningRepIsSelected = choices.contains(SeverityCode.WARNING.getCode());
+        final boolean infoRepIsSelected = choices.contains(SeverityCode.INFO.getCode());
+        final boolean fineRepIsSelected = choices.contains(SeverityCode.FINE.getCode());
+        final boolean severePopupIsSelected = choices2.contains(SeverityCode.SEVERE.getCode());
+        final boolean warningPopupIsSelected = choices2.contains(SeverityCode.WARNING.getCode());
+        final boolean infoPopupIsSelected = choices2.contains(SeverityCode.INFO.getCode());
+        final boolean finePopupIsSelected = choices2.contains(SeverityCode.FINE.getCode());
+        
+        final String severeReportBorderShade = severeRepIsSelected ? severeFill : INACTIVE_BACKGROUND;
+        final String warningReportBorderShade = warningRepIsSelected ? warningFill : INACTIVE_BACKGROUND;
+        final String infoReportBorderShade = infoRepIsSelected ? infoFill : INACTIVE_BACKGROUND;
+        final String fineReportBorderShade = fineRepIsSelected ? fineFill : INACTIVE_BACKGROUND;
+        final String severePopupBorderShade = severePopupIsSelected ? severeFill : INACTIVE_BACKGROUND;
+        final String warningPopupBorderShade = warningPopupIsSelected ? warningFill : INACTIVE_BACKGROUND;
+        final String infoPopupBorderShade = infoPopupIsSelected ? infoFill : INACTIVE_BACKGROUND;
+        final String finePopupBorderShade = finePopupIsSelected ? fineFill : INACTIVE_BACKGROUND;
+
+        
+        updateSettingsIcon(severeReportFilter, severeReportBorderShade, severeRepIsSelected ? severeBorder : severeFill);
+        updateSettingsIcon(warnReportFilter, warningReportBorderShade, warningRepIsSelected ? warningBorder : warningFill);
+        updateSettingsIcon(infoReportFilter, infoReportBorderShade, infoRepIsSelected ? infoBorder : infoFill);
+        updateSettingsIcon(fineReportFilter, fineReportBorderShade, fineRepIsSelected ? fineBorder : fineFill);
+        updateSettingsIcon(severePopupAllowed, severePopupBorderShade, severePopupIsSelected ? severeBorder : severeFill);
+        updateSettingsIcon(warnPopupAllowed, warningPopupBorderShade, warningPopupIsSelected ? warningBorder : warningFill);
+        updateSettingsIcon(infoPopupAllowed, infoPopupBorderShade, infoPopupIsSelected ? infoBorder : infoFill);
+        updateSettingsIcon(finePopupAllowed, finePopupBorderShade, finePopupIsSelected ? fineBorder : fineFill);
+        
         updateFilterData();
     }
 
     public void updateFilterData() {
+        
+
+        List<String> choices = new ArrayList<>();
+        List<String> popupChoices = new ArrayList<>();        
+        if (getParams().hasParameter(REPORT_SETTINGS_PARAMETER_ID)) {
+            MultiChoiceParameterValue multiChoiceValue = getParams().getMultiChoiceValue(REPORT_SETTINGS_PARAMETER_ID);
+            choices = multiChoiceValue.getChoices();
```

Comment:
Is the value of `choices` supposed to be used somewhere?
If not, this block can be removed, as well as the variable definition.